### ISSUE TITLE
[Help]: Where is located the log file?

### ISSUE THREAD

**Issue by barbas66** [JaKooLit/Hyprland-Dots#226]
Hi and congratulations for the good work.

When I start hyprland I see some error messages, but the notifications promptly disappear.

Is there any log file that reports the errors?

Thanks  a lot.

**Comment by JaKooLit** [comment]
https://wiki.hyprland.org/Crashes-and-Bugs/

first lines maybe?

**Comment by barbas66** [comment]
OK. Thanks!

**Comment by barbas66** [comment]
Just so everybody knows, the problem arose because a plugin needed an update. How I found out:

1. There wasn't a crash log file as hyprland didn't crash.

2. I located the log file in:

`/tmp/hypr/fe7b748eb668136dd0558b7c8279bfcd7ab4d759_1714609481/hyprland.log
`
3. The log file was barebone, I had to enable an extended log file by inserting the following lines

```
debug {
disable_logs=false
} 

```
 in the hyprland conf file.

4. Turns out the only relevant information given in said log file was: 

`[ERR] Invalid dispatcher!
`
5. Bottom line: I finally managed to read the red notifications appearing as hyprland was started - this time (maybe because debug was enabled?)  they didn't disappear - and they referred to the lines in the .conf file which were causing problems.
